### ISSUE TITLE
plot psychological continuum for methods that assume transitive global preferences

### ISSUE THREAD

**Issue by istvanp1** [skpref/skpref#14]
This is a simple visualisation that I like very much, because not only does it give an idea for the rank but also of how much gap there is between the preferences in the options. I have seen it on two occasions both times on the same data. Originally in Thurstone's paper [The Method of Paired Comparisons for Social Values](https://psycnet.apa.org/record/2006-01643-007) 
![image](https://user-images.githubusercontent.com/25740842/70389168-9f428a80-19b3-11ea-9095-0beab921b2ed.png)
And also in [Maystre's thesis](https://infoscience.epfl.ch/record/255399). This is the simpler representation so I think we should go with this one first 
![image](https://user-images.githubusercontent.com/25740842/70389190-ffd1c780-19b3-11ea-9ace-e247bfccfa8b.png)

**Comment by fkiraly** [comment]
I think the Maystre suggestion is more difficult to read and causes potential problems due to text overlapping. Thurstone's suggestion I find more sensible - the difference being what in modern terminology one calls "leader lines", i.e., lines that connect the words to the dot which they annotate.
The best of two worlds is perhaps a leader line annotated vertical scale, with vertical text, or horizontal text with leader lines such as in the critical difference plots.

**Comment by fkiraly** [comment]
Actually, come to think of it: if you add in significant/critical rank differences, as the more common ranking GLM give you, what you are left with *is* a critical difference plot.

**Comment by istvanp1** [comment]
That's true, though I haven't managed to quickly find any example in matplotlib that can annotate a scatterplot with leader lines. But there are some [examples with a doughnut chart](https://matplotlib.org/3.1.1/gallery/pie_and_polar_charts/pie_and_donut_labels.html). 
And there is also the common case of [labelling points on the scatterplot](https://www.pythonmembers.club/2018/05/08/matplotlib-scatter-plot-annotate-set-text-at-label-each-point/). We can try to combine these two somehow.
